x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/fix_logs_step/fix_logs_step.test.tsx

```diff
@@ -151,4 +148,105 @@ describe('Overview - Fix deprecation logs step', () => {
       expect(find('viewDiscoverLogs').props().href).toBe('/discover/logs');
     });
   });
+
+  describe('Step 3 - Resolve log issues', () => {
```

Comment:
Should we add a test to verify this step only renders if log indexing is enabled?